_includes/head/custom.html

```diff
@@ -9,7 +9,8 @@
 
 {% if site.baseurl %}
 <div id="banner">
-    This is the documentation for an unreleased version of enmeshed, which is currently in a non-stable beta status. Not what you are looking for?
+    This is the documentation for an unreleased version of enmeshed, which is currently in a non-stable beta status. Not
+    what you are looking for?
     <a href="/">Head over to the current version.</a>
 </div>
-{% endif %}
+{% endif %}
```

Comment:
please install the prettier extension in your vscode or run prettier, newline also missing here.